src/Service/DigitalPostHelper.php

```diff
@@ -72,6 +77,106 @@ public function sendDigitalPost(string $cpr, string $name, Address $address, str
         return $result;
     }
 
+    public function createDigitalPost(Document $document, string $subject, string $entityType, Uuid $entityId, array $digitalPostAttachments, array $digitalPostRecipients): void
+    {
+        $digitalPosts = [];
+
+        // Handle attachments and add extra digital post entities if necessary due to size or number of attachments
+        // Size limit for attachments and main document combined is 80mb
+        // The max number of attachments per digital post is 10
+        $documentPath = $this->getFilePath($document);
+        $currentSizeOfDigitalPost = $mainDocumentSize = filesize($documentPath);
+        $currentNumberOfAttachments = 0;
+        $sizeLimit = 80 * (10 ** 6);
+        $attachmentLimit = 10;
```

Comment:
These should be configurable. References to why the size limit i 80 (MB) and why the total number of attachments is 10 will also be useful (but it may be hard to point to this in the Digital post documentation).

Reply:
Made both configurable and added references.